sql/moz-fx-data-shared-prod/mozilla_vpn_derived/funnel_product_page_to_subscribed_v1/query.sql

```diff
@@ -107,15 +113,51 @@ flows AS (
     LOGICAL_OR(event_type = "fxa_subscribe_coupon - fail") AS subscribe_coupon_fail,
     LOGICAL_OR(event_type = "fxa_subscribe_coupon - success") AS subscribe_coupon_success,
   FROM
-    mozdata.firefox_accounts.fxa_content_auth_stdout_events
+    events
   WHERE
-    IF(@date IS NULL, DATE(`timestamp`) < CURRENT_DATE, DATE(`timestamp`) = @date)
+    IF(
+      @date IS NULL,
+      -- The fully FxA-based funnel for VPN was enabled on 2021-08-25.
+      (partition_date >= "2021-08-25" AND partition_date < CURRENT_DATE),
```

Comment:
@YeonjooSmith I want to double check that limiting this to >= 2021-08-25 won't cause any issues for how this is being used in reports.

(we could allow previous data, but the top-of-funnel `vpn_site_hits` calculation only works from when the fully FxA-based funnel was enabled on 2021-08-25)

Reply:
Yes the limiting dates to >=>= 2021-08-25 will be okay. Thank you for confirming!